playwright.config.ts

```
@@ -3,8 +3,29 @@ import dotenv, { DotenvConfigOutput } from 'dotenv';
+  process.env.NEXT_PUBLIC_PROD_CONTAINER_API_URL ||
+  process.env.NEXT_PUBLIC_WEBSITE_URL ||
+  'http://127.0.0.1:3001';
+
+const commonSettings:CommonSettings = {
+  ignoreHTTPSErrors: true,
+  baseURL,
+  trace: 'on-first-retry' as const,
+  extraHTTPHeaders: process.env.NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME
+    ? {
+        [`aws-cf-cd-${process.env.NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME}`]:
+          process.env.NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)